FW/FreeRTOS/MessageBuffer.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef _FREERTOS_MESSAGE_BUFFER_HPP
+#define _FREERTOS_MESSAGE_BUFFER_HPP
+
+
+#include "FreeRTOS.h"
+#include "message_buffer.h"
+
+
+namespace FreeRTOS
```

Comment:
Usually you don't indent code that is in a namespace, similarly to how you don't indent code inside an include guard, since it just adds an indent tax of 1 to everything.